src/Controller/CarController.php

```diff
@@ -107,19 +109,12 @@ public function getUserCars(
         ]
     )]
     public function index(): Response
-        { 
-        $url = $this->apiHost . '/api/users/'. $this->security->getUser()->getId() .'/cars';
-        $response = $this->httpClient->request('GET', $url);
-        $content = $response->getContent(false);
-        $rawData = json_decode($content, true);
-        if (json_last_error() !== JSON_ERROR_NONE) {
-        $rawData = [];
-        }
-
+    {
+        $rawData = $this->carService->getAllCarsForUser($this->security->getUser()->getId());
         return $this->render('car/index.html.twig', [
-        'cars' => $rawData,
+            'cars' => $rawData,
```

Comment:
instead of having $rawData variable, put directly here:
'cars' =>  $this->carService->getAllCarsForUser($this->security->getUser()->getId()),